modules/tensor_mechanics/include/materials/ComputeThermalExpansionEigenstrain.h

```diff
@@ -28,8 +29,8 @@ class ComputeThermalExpansionEigenstrain : public ComputeEigenstrainBase
   virtual void computeQpEigenstrain();
 
   const VariableValue & _temperature;
+  MaterialProperty<RankTwoTensor> & _dthermal_expansion_dT;
```

Comment:
In this case, it really is the thermal expansion (rather than some arbitrary eigenstrain).  I'm happy to change that name, though.